project/roadmap.md

```diff
@@ -4,61 +4,71 @@
 
 If you are interested in collaborating on any of the roadmap features or other features in the project, please mail [Mbed TLS](https://lists.trustedfirmware.org/mailman/listinfo/mbed-tls) or [PSA Crypto](https://lists.trustedfirmware.org/mailman/listinfo/psa-crypto) mailing lists.
 
-**Completed**
-
-* Merge Mbed Crypto repository into Mbed TLS. Mbed Crypto still available and under active development in Mbed TLS repository
-* [Unified PSA Driver Interface](https://github.com/orgs/ARMmbed/projects/18#column-15836331) - API Design and initial support
-* [PSA Crypto] [PSA Crypto API implementation v1.0 Specification - Phase1](https://github.com/orgs/ARMmbed/projects/18#column-15836285)
-* Lucky13 Security Improvements 
-* DTLS-SRTP 
-* [PSA Crypto]​ ECJPAKE API Design  ​
-* [PSA Crypto]​ PBKDF2 API  Design 
-* [PSA Crypto]​ EdDSA API  Design 
-* [PSA Crypto]​ M-AEAD Implementation
-* [PSA Crypto] [Support Missing PSA Crypto v1.0 APIs supported in MbedCrypto](https://github.com/orgs/ARMmbed/projects/18#column-15836299)
-* [Mbed TLS] [New 2.x LTS](https://github.com/orgs/ARMmbed/projects/18#column-15836286)
-* [Mbed TLS] [​SHA256/512 - Neon Optimization](https://github.com/orgs/ARMmbed/projects/18#column-16274498)
-* [Mbed TLS] [Open CI - Initial](https://ci.trustedfirmware.org/view/Mbed-TLS/)
-* [Mbed TLS] [TLS/X.509​ - Use PSA Crypto APIs Fully​ - Phase1](https://github.com/orgs/ARMmbed/projects/18#column-15836318)
-* [Mbed TLS] [TLS1.3 MVP](https://github.com/orgs/ARMmbed/projects/18#column-15836288)
-* [Mbed TLS] [TLS1.3 server side](https://github.com/orgs/Mbed-TLS/projects/1#column-17950131)
-* [Mbed TLS] [Mbed TLS3.2](https://github.com/orgs/Mbed-TLS/projects/1#column-18338314)
-* [Mbed TLS] [TLS1.3 PSK](https://github.com/orgs/Mbed-TLS/projects/1#column-17950145)
-* [Mbed TLS] [TLS/X.509​ - Use PSA Long term secret isolation](https://github.com/orgs/Mbed-TLS/projects/1#column-183383222)
-* [Mbed TLS] [Use PSA: code size optimisation via PSA_CRYPTO_CONFIG - part1: symmetric](https://github.com/orgs/Mbed-TLS/projects/1#column-18883163)
-* [Mbed TLS] [Connection ID DTLS1.2](https://github.com/Mbed-TLS/mbedtls/pull/6264)
-
-**2023 CQ1 (In Development)**
+**2023 CQ2 (In Development)**
+* [Mbed TLS] [Code size optimisation (driver only build - ECC - Initial)](https://github.com/orgs/Mbed-TLS/projects/1#column-18986128)
+* [PSA Crypto] Memory Optimizations - Phase1
```

Comment:
When I see “memory optimizations” out of context, I think of RAM optimization. But our focus is on code size, so I think this and other similar lines should say “code size optimizations”.

Reply:
Added (code size) for the Memory optimization entries